frontend/src/components/IstioStatus/IstioStatus.tsx

```diff
@@ -40,7 +39,6 @@ type ReduxStateProps = {
 };
 
 type ReduxDispatchProps = {
-  refreshNamespaces: () => void;
```

Comment:
Above, at line 37, you need to remove `namespaces?: Namespace[]`, because you removed it from the `mapStateToProps`.  It's good you removed it, we now just need to clean up a few things.